onnxruntime/core/session/custom_ops.cc

```diff
@@ -125,18 +125,37 @@ common::Status CreateCustomRegistry(const std::vector<OrtCustomOpDomain*>& op_do
       size_t type_id_counter = 0;
       auto input_count = op->GetInputTypeCount(op);
       for (size_t i = 0; i < input_count; i++) {
+        onnx::OpSchema::FormalParameterOption option = onnx::OpSchema::FormalParameterOption::Single;
+
+        // only since the ORT API version 7 and onwards will users have defined GetInputCharacteristic() and GetOutputCharacteristic()
```

Comment:
nit: slightly confusing to say the 'users' will have defined the methods. it's more that the OrtCustomOp interface won't have them isn't it? also need to update the function names in the comment to add 'CustomOp' after the 'Get'.

Reply:
Fixed the wording of the comment and the updated the function name